analysis/study_definition.py

```diff
@@ -471,12 +690,44 @@
     ),
 )
 
-# calculate crude mortality rate
 measures = [
+    # calculate crude mortality rate
+    Measure(
+        id = "crude_mortality_rate",
+        numerator = "died_ons_covid_flag_any",
+        denominator = "population",
+        group_by = "population",
+    ),
+    # calculate rates in age groups (for females and males seperately)
+    Measure(
+        id = "age_mortality_rate",
+        numerator = "died_ons_covid_flag_any",
+        denominator = "population",
+        group_by = ["sex", "agegroup"],
+    ),
+    # calculate rates in females/males
     Measure(
-        id="crude_mortality_rate",
-        numerator="died_ons_covid_flag_any",
-        denominator="population",
-        group_by="population",
+        id = "sex_mortality_rate",
+        numerator = "died_ons_covid_flag_any",
+        denominator = "population",
+        group_by = ["agegroup_std", "sex"],
     ),
-]
\ No newline at end of file
+]
+
+for demographic in demographics_list:
```

Comment:
Nice!

Note: You could just call "m1" and "m2" "m" as it is only defined in the scope of the for loop